core/oracle/token_price_manager.go

```diff
@@ -106,29 +127,49 @@ func (m *defaultTokenPriceManager) getRequest(tokenList []string) *http.Request
 	return req
 }
 
-func (m *defaultTokenPriceManager) getTokenPrices(tokenList []string) ([]*types.TokenPrice, error) {
-	req := m.getRequest(tokenList)
-	data, err := m.networkHttp.Get(req)
-	if err != nil {
-		return nil, err
-	}
+func (m *defaultTokenPriceManager) getResponse(tokenList []string) ([]*types.TokenPrice, error) {
+	tokenPrices := make([]*types.TokenPrice, 0)
+	tokensNotAvailable := make([]string, 0)
+	tokens := m.cfg.Tokens
+
+	for _, token := range tokenList {
+		address := tokens[strings.ToLower(token)].Address
+		tokenPrice, err := m.uniswapManager.GetPriceFromUniswap(address)
+
+		if err != nil {
+			tokenPrice, err = m.sushiswapManager.GetPriceFromSushiswap(address)
+			if err != nil {
+				tokensNotAvailable = append(tokensNotAvailable, token)
+			}
+		}
+		tokenPrices = append(tokenPrices, tokenPrice)
 
-	response := &Response{}
-	err = json.Unmarshal(data, &response)
-	if err != nil {
-		return nil, err
 	}
+	if len(tokensNotAvailable) > 0 {
```

Comment:
Add comment here to indicate that this is getting token price from CoinmarketCap